### PR TITLE
Andreea

### PR DESCRIPTION
**Pull request by AirinB**

### CODE REVIEW

File: core/src/main/java/com/rim/IceField/Shovel.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.rim.IceField;
+
+public class Shovel extends ItemBase {
+    public Shovel(String tag) {
```
Comment (RainZoneO2) [inline core/src/main/java/com/rim/IceField/Shovel.java:4]: The shovel's tag here should just be "Shovel" automatically.

File: core/src/main/java/com/rim/IceField/Shovel.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        super(tag);
+    }
+
+    @Override
+    public boolean useItem(PlayerBase player) {
+        return false;
+    }
```
Comment (RainZoneO2) [inline core/src/main/java/com/rim/IceField/Shovel.java:11]: How does this work?

File: core/src/main/java/com/rim/IceField/ItemBase.java [hunk 3]
```diff
@@ -18,6 +18,8 @@ public ItemBase(String tag) {
         this.active = false;  //the item that does not belog to the inventory, cannnot be used
     }
 
+    public abstract boolean useItem(PlayerBase player);
+
```
Comment (RainZoneO2) [inline core/src/main/java/com/rim/IceField/ItemBase.java:22]: Why is player in the parameter? The use item can be called by the player, no need to have the player in the parameter

**Review by RainZoneO2 — CHANGES_REQUESTED** [review]
Check the comments I put, thanks :)

File: core/src/main/java/com/rim/IceField/Shovel.java (hunk 1 above)
Comment (AirinB) [reply core/src/main/java/com/rim/IceField/Shovel.java:4]: You are right

File: core/src/main/java/com/rim/IceField/Shovel.java (hunk 2 above)
Comment (AirinB) [reply core/src/main/java/com/rim/IceField/Shovel.java:11]: I did not complete this one

File: core/src/main/java/com/rim/IceField/ItemBase.java (hunk 3 above)
Comment (AirinB) [reply core/src/main/java/com/rim/IceField/ItemBase.java:22]: Because I need it in the method implementation in the derived classes

**Review by RainZoneO2 — APPROVED** [review]

**Pull request merged**